frontend/src/components/Toolbar/Toolbar.js

```diff
@@ -54,6 +54,14 @@ const tools = [
     value: 'comment',
     icon: <MessageSquare />,
     animation: commentAnimation
+  },
+  {
+    label: 'Template tool',
+    hotkey: 'J',
+    description: 'Add templates to your automaton',
+    value: 'template',
+    icon: <Star />,
+    animation: commentAnimation
```

Comment:
Would be good if it had its own unique animation when you hover over the tool as I found this really useful in understanding how to use the other tools. I'm not familiar with how it's able to do that though I've never seen JSON files used in that way (see the JSON files in the animation folder in the Toolbar directory), might be best to consult group 1 on how they did this if neither you nor Jake know (assuming they implemented this animation feature).